control/controlclient/direct.go

```diff
@@ -1220,3 +1228,16 @@ func ipForwardingBroken(routes []netaddr.IPPrefix) bool {
 
 	return false
 }
+
+func getNodeDisplayName(node *tailcfg.Node) string {
```

Comment:
there may be a better place for this function than direct.go

Reply:
move to a method on Node, like DefaultDisplayName?